### PR TITLE
fix(core-flows): Allow return requests with no shipping

### PR DESCRIPTION
**Pull request by olivermrbl**
Closes CC-265

### CODE REVIEW

File: packages/core/core-flows/src/order/workflows/return/confirm-return-request.ts [hunk 1]
```diff
@@ -106,10 +106,6 @@ function prepareFulfillmentData({
 }
 
 function extractReturnShippingOptionId({ orderPreview, orderReturn }) {
-  if (!orderPreview.shipping_methods?.length) {
```
Comment (olivermrbl) [inline packages/core/core-flows/src/order/workflows/return/confirm-return-request.ts:109]: We can't rely on this check, as the preview also contains the regular shipping methods of the order.

Comment (carlos-r-l-rodrigues) [reply packages/core/core-flows/src/order/workflows/return/confirm-return-request.ts:109]: this is just skipping if nothing exist. not important, we could just modify the loop as `of orderPreview.shipping_methods ?? []`
Inside the loop, we check for the `actions` on each method and get the correct one.

Comment (olivermrbl) [reply packages/core/core-flows/src/order/workflows/return/confirm-return-request.ts:109]: Yeah, I can do that before I merge 👍

File: packages/modules/order/src/services/actions/receive-return.ts [hunk 2]
```diff
@@ -50,6 +50,9 @@ function updateReturnItems(returnEntry, items) {
       const data = items.find((i) => i.details.reference_id === item.item_id)
       if (!data) return
 
+      console.log("ITEM: ", item)
```
Comment (carlos-r-l-rodrigues) [inline packages/modules/order/src/services/actions/receive-return.ts:53]: @olivermrbl Oups...

Comment (olivermrbl) [reply packages/modules/order/src/services/actions/receive-return.ts:53]: Damn my bad :'((

**Pull request merged**